### PR TITLE
TYP: pandas/plotting annotations from pandas-stubs

### PR DESCRIPTION
**Pull request by twoertwein**
Copied annotations from pandas-stubs when pandas has no annotations

### CODE REVIEW

**Review by simonjayhawkins — APPROVED** [review]
Thanks @twoertwein cc @Dr-Irv

File: pandas/core/arrays/arrow/array.py [hunk 1]
```diff
@@ -444,7 +444,7 @@ def dropna(self: ArrowExtensionArrayT) -> ArrowExtensionArrayT:
         else:
             return type(self)(pc.drop_null(self._data))
 
-    def isin(self: ArrowExtensionArrayT, values) -> npt.NDArray[np.bool_]:
+    def isin(self, values) -> npt.NDArray[np.bool_]:
```
Comment (Dr-Irv) [inline pandas/core/arrays/arrow/array.py:447]: Why make this change?

File: pandas/plotting/_misc.py [hunk 2]
```diff
@@ -363,17 +388,17 @@ def bootstrap_plot(series, fig=None, size=50, samples=500, **kwds) -> Figure:
...
-    color=None,
-    use_columns=False,
-    xticks=None,
+    frame: DataFrame,
+    class_column: str,
+    cols: list[str] | None = None,
+    ax: Axes | None = None,
+    color: list[str] | tuple[str, ...] | None = None,
+    use_columns: bool = False,
+    xticks: Sequence | None = None,
```
Comment (Dr-Irv) [inline pandas/plotting/_misc.py:397]: based on the docs, this should be `list | tuple | None`

**Review by Dr-Irv — CHANGES_REQUESTED** [review]

File: pandas/core/arrays/arrow/array.py (hunk 1 above)
Comment (twoertwein) [reply pandas/core/arrays/arrow/array.py:447]: This TypeVar is useless (my local pyright version complained about it). I'll bump pyright on the CI

File: pandas/plotting/_misc.py (hunk 2 above)
Comment (twoertwein) [reply pandas/plotting/_misc.py]: changed

**Review by Dr-Irv — APPROVED** [review]
LGTM

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @twoertwein

**Pull request merged**